### ISSUE TITLE
Allow a teacher/admin to see the list of lessons with controls

### ISSUE THREAD

**Issue by amellie** [Ali-Jahankah/bugsquashers-edu-app#27]
## Dependency

Depends on #21 and #16 

## Story
**As a** Teacher or an Admin
**I Want** to see the list of existing lessons
**So That** I can see review these lessons and perform actions (create/update/etc)

## Acceptance Criteria
- The page should only be accessible by a teacher/admin
- The page can be accessed from the module list page created in #21. You need to modify the module list page so that the lesson numbers are linked to this page
- The page should contain the module name as well as a table containing these columns:
-- Name (of the lesson)
-- Type
-- Action (which contains: Edit and Delete button)
- A Create button should be added to the top of the table. When the button is clicked, it should show the page that has been done in #16 
- Edit and Delete buttons do not do anything when clicked (this will be handled in another task, so please leave it as is)
